oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java

```diff
@@ -151,6 +217,43 @@ private Object createOracleTimestampWithTimeZone(Connection connection, String t
     }
   }
 
+  /**
+   * Creates an instance of 'oracle.sql.TIMESTAMPLTZ' which corresponds to the specified timestamp with local time zone.
+   * @param connection sql connection.
+   * @param timestampString timestamp with local time zone string, such as "2019-07-15 15:57:46.65 GMT".
+   * @return instance of 'oracle.sql.TIMESTAMPLTZ' which corresponds to the specified timestamp with local time zone
+   * string.
+   */
+  private Object createOracleTimestampWithLocalTimeZone(Connection connection, String timestampString) {
+    try {
+      ClassLoader classLoader = connection.getClass().getClassLoader();
+      Class<?> timestampLTZClass = classLoader.loadClass("oracle.sql.TIMESTAMPLTZ");
+      return timestampLTZClass.getConstructor(Connection.class, String.class).newInstance(connection, timestampString);
+    } catch (ClassNotFoundException e) {
+      throw new InvalidStageException("Unable to load 'oracle.sql.TIMESTAMPLTZ'.", e);
+    } catch (InstantiationException | InvocationTargetException | NoSuchMethodException | IllegalAccessException e) {
+      throw new InvalidStageException("Unable to instantiate 'oracle.sql.TIMESTAMPLTZ'.", e);
+    }
+  }
+
+  /**
+   * Creates an instance of 'oracle.sql.TIMESTAMP' which corresponds to the specified timestamp with local time zone.
```

Comment:
Is this comment correct? Looks like oracle TIMESTAMP does not contain timezone information.

```
Using the example format, specify TIMESTAMP as a literal as follows:
TIMESTAMP '1997-01-31 09:26:50.12'
```

Reply:
Agreed.